unstructured/ingest/interfaces.py

```diff
@@ -61,6 +63,20 @@ class ReadConfig(BaseConfig):
     download_only: bool = False
 
 
+@dataclass
+class EmbeddingConfig(BaseConfig):
+    api_key: str
+    model_name: t.Optional[str] = None
+
+    def get_embedder(self) -> BaseEmbeddingEncoder:
+        kwargs = {
+            "api_key": self.api_key,
+        }
+        if self.model_name:
+            kwargs["model_name"] = self.model_name
+        return OpenAIEmbeddingEncoder(**kwargs)
```

Comment:
this always returns an instance of OpenAIEmbeddingEncoder? would have assumed the encoder was determined by a parameter? or is this placeholder until we have more than one? (if so, probably good to mark a TODO on this)